src/components/ArticleCard.jsx

```diff
@@ -40,8 +61,17 @@ function IndividualArticleCard() {
           <p>{article.body}</p>
           <p>Author: {article.author}</p>
           <p>Published: {article.created_at}</p>
+          <p>Votes: {currentVotes}</p> 
+
+          <button className={`vote-button ${userVote === 'up' ? 'upvoted' : ''}`} onClick={() => handleVote('up')}>
+            {userVote === 'up' ? '👍' : 'Upvote'}
+          </button>
+          <button className={`vote-button ${userVote === 'down' ? 'downvoted' : ''}`} onClick={() => handleVote('down')}>
+            {userVote === 'down' ? '👎' : 'Downvote'}
+          </button>
+
```

Comment:
Please only display 'upvote' and 'downvote' buttons when the current vote is set to the original one.